core/sql/optimizer/SearchKey.h

```diff
@@ -1042,6 +1045,10 @@ class HbaseSearchKey : public SearchKey
    ValueIdSet requiredOutputColumns_;
 
    ValueIdSet nonKeyPreds_;
+
+   // is a constant folded pred that returns FALSE always. SK has 
+   // min/max for begin/end reversed.
+   NABoolean isFalsePred_;
```

Comment:
In the SearchKey begin and end keys are already reversed as we want for the isAFalseConstant predicate Please see SearchKey.cpp around Line 147.